### PR TITLE
Convesation Tester Documentation

### PR DESCRIPTION
**Pull request by jgonzalezal**
Convesation Tester Documentation

### CODE REVIEW

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+## Invalid Private Text Body
+## Description
+This error happens when the bot's response has an invalid value in its private text message and the vendor is unable to parse the content. It can be a missing property or a wrong parameter or value.
+
+## Invalid AI Vendor Response
+## Description
+This error happens when there was not possible to get any response from the vendor.
+
+#### Errors Solution
```
Comment (fahimmahmoodmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:57]: Probably it is better to provide Error and Solution for each of the errors to that relevant document link in TPB. Likes for example in Invalid Delay Body for Watson refer the link to the Delay section of the Watson place. You don't have to make a separate heading named solution. You can simply after describing the error lead the user to the relevant documentation

**Review by fahimmahmoodmir — CHANGES_REQUESTED** [review]

File: pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+When testing your bot and you get an error in the response, you will be able to see what was the JSON object that produced the error in your bot response. By hovering on the last alert you will see a tooltip that will give you the possible solutions for your error. See the image below.
+
+
+<img style="width:800px" src="img/botconnectordashboard/error_response.png">
+
+### Errors that can be found per vendor
+
+### Dialog Flow V2
+
+## Dialog Flow V2 Invalid Action Body
```
Comment (fahimmahmoodmir) [inline pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:29]: Are you going to show the exact same text in conversation tester UI if this error happened? Just wanted to make sure that users are able to make text connection to what they see in the conversation tester UI

Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:29]: It is not exactly the same text. For example when I have an Action body error, the error will say 
Unable to parse action due to the invalid body

Comment (fahimmahmoodmir) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:29]: Ok so you need to update it in UI as well in such a way that user know and able to make the link

Comment (jgonzalezal) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:29]: @fahimmahmoodmir  In the UI I have the error message, for example "Unable to parse action due to the invalid body", that is the description the user needs in the UI, here in the documentation I have the error type as title and the explanation. I don't think I have to change the error message in the UI.

Comment (fahimmahmoodmir) [reply pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md:29]: Ok if you think the user will be able to make the link between UI message and the documentation then its fine

**Pull request merged**